src/mainwindow.cpp

```diff
@@ -921,11 +921,28 @@ void ShowSummary(TreeModel* model, QWidget* parent)
         std::unique_ptr<CounterMap> SubCounters;
     };
 
+    QString beginQueryEventKey = "begin-query";
+    bool foundAQuery = false;
```

Comment:
I think this variable can be deleted; and you could `break;` the loop whenever you find a query in the loop below.

Reply:
yes, good point. i'll change it.

Reply:
changed.